application/controllers/TemplateController.php

```diff
@@ -6,10 +6,13 @@
 
 use DateTime;
 use GuzzleHttp\Psr7\ServerRequest;
+use Icinga\Exception\MissingParameterException;
```

Comment:
Some leftovers from your previous commit?

Reply:
Yes, most of this is obsolete since the TemplateForm, etc. refactor PR.